mobile_verifier/src/heartbeats/mod.rs

```diff
@@ -383,7 +382,7 @@ impl ValidatedHeartbeat {
                 proto::Heartbeat {
                     cbsd_id: self.heartbeat.cbsd_id.clone().unwrap_or_default(),
                     pub_key: self.heartbeat.hotspot_key.as_ref().into(),
-                    reward_multiplier: self.cell_type.reward_weight().to_f32().unwrap_or_default(),
+                    reward_multiplier: 1.0,
```

Comment:
can this be deprecated and then just ignored? if not, please comment "this is a legacy field that's been superseded by algorithmic coverage weights" or something

Reply:
We are planning on deprecating this field at some point